data-structures-stacks/max-value.js

```diff
@@ -0,0 +1,14 @@
+/* exported maxValue */
+
+function maxValue(stack) {
+  let max = -Infinity;
+  if (stack.peek() !== undefined) {
```

Comment:
You don't need the `if` here since you are already doing the same check in the `while` loop. If it starts as `undefined` the `while` loop code block will never run.